mails/themes/classic/core/order_conf.html.twig

```diff
@@ -146,6 +146,7 @@
             </table>
           </td>
         </tr>
+        {% if giftWrapping == 1 %}
```

Comment:
You put the condition on the shipping line.

Reply:
Oh my, you're right!